core/controllers/base_test.py

```diff
@@ -272,7 +272,7 @@ def setUp(self):
 
     def test_jinja_autoescaping(self):
```

Comment:
I'm a little uncertain why this test still passes. It used to be doing interpolation but clearly isn't any more because we are removing a constant. Do you know what it's currently detecting?

Reply:
It's still detecting a constant.
I used DEV_MODE instead and I added DEV_MODE to `core/templates/dev/head/pages/tests/jinja_escaping.html`

Reply:
I see, OK. I think the form_url variable name is quite misleading, then. Can we change it to dangerous_field_contents or something like that?

And maybe let's just get rid of text etc. that's not relevant on the test page (assuming it's used for this test and nothing else).

Reply:
Okay.
I checked and it's only used for this test.